codefordenver/cdphe-accessible-covid19
# Keys to lowercase2

#### What does this PR do?

Adds a function into utilities.js to convert "Date" keys in the API data to lowercase ("date"), updates the sortsObjectsByDate function in utilities.js to handle the new data key; updates the cleanDailyStats function in apiClient.js to convert the keys to lowercase; updates XYGraph to use the new "date" key instead of the dateKey variable and function

#### Where should the reviewer start?

src/utils-> utilities.js -> apiClient.js -> src/components/Graphs/XYGraph/XYGraph.js

#### How should this be manually tested?

npm start, make sure all graphs are still rendering as they should (currently, I think only the "tested" page is utilizing the XYGraph)

#### Any background context you want to provide?

N/A

#### What are the relevant tickets?

Lowercase all data keys on consumption #32

#### Questions:

N/A
